fragments/install/cephadm/add-mds_using_placement.yaml

```diff
@@ -0,0 +1,14 @@
+tests:
```

Comment:
can we rename this fragment to `add_mds_using_nodes`?

Reply:
Yes pragdeesh
I will do that.